ramsey/uuid-doctrine/1.3/config/packages/ramsey_uuid_doctrine.yaml

```diff
@@ -0,0 +1,4 @@
+doctrine:
```

Comment:
I see one issue here: `ramsey/uuid-doctrine` does not require `doctrine/doctrine-bundle`. So if one installs the UUID package without requiring DoctrineBundle too, this part will lead to errors. Are we okay with that?

Reply:
but why else would I use it in a symfony project?